index.html

```diff
@@ -0,0 +1,666 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Maven+Pro:wght@600&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Heebo:wght@300&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/9a2f12287f.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+    <title>TNW</title>
+</head>
+
+<body>
+    <header>
+        <nav class="nav1">
+            <ul>
+                <li>News</li>
+                <li>Events <i class="fas fa-angle-down"></i> </li>
+                <li>Business <i class="fas fa-angle-down"></i> </li>
+                <li>AMAs</li>
+                <li>Spaces</li>
+                <li>Terms & Conditions</li>
+            </ul>
+            <ul>
+                <li>About</li>
+                <li>Advertise</li>
+                <li>Jobs</li>
+                <li>Contact</li>
```

Comment:
- [ ] There should be a TNW logo on the medium screen size (left size)

| Yours | Original |
|:---:|:---:|
| ![image](https://user-images.githubusercontent.com/61479648/88698902-09dc3e00-d0dd-11ea-9566-73f947ee4d32.png) | ![image](https://user-images.githubusercontent.com/61479648/88698954-1d87a480-d0dd-11ea-8e90-eeadd2685ca3.png) |

- [ ] The height should be smaller on the mobile screen size


| Yours | Original |
|:---:|:---:|
| ![image](https://user-images.githubusercontent.com/61479648/88699087-4740cb80-d0dd-11ea-9590-d3a998ec0fa8.png) | ![image](https://user-images.githubusercontent.com/61479648/88699109-4f007000-d0dd-11ea-91a1-77b35a8b2334.png) |